scripts/tests/localsetup_tests.py

```diff
@@ -808,6 +813,17 @@ def test_start_6_3_default(self):
         """)  # noqa: 501
         self.assertDictEqual(got, want)
 
+    def test_version_options(self):
```

Comment:
It is fine, there is a simple way to test the same by using `render` instead of the `LocalSetup`, something like this

```
def test_opbeans_java_service_version(self):
  opbeans = OpbeansJava(opbeans_java_service_version="1.2.3").render()["opbeans-java"]
  self.assertEqual("1.2.3", opbeans["environment"]["ELASTIC_APM_SERVICE_VERSION"])
```